### PR TITLE
Bug Fix | Workplace Permissions & References

### PR DESCRIPTION
**Pull request by ChazUK**

### CODE REVIEW

**Comment by ChazUK** [comment]
[(3) Bulk Upload - Local Identifiers - Remove ability to update References for Workplaces that are not owned by Parent](https://trello.com/c/LYzUh233/41-3-bulk-upload-local-identifiers-remove-ability-to-update-references-for-workplaces-that-are-not-owned-by-parent)

File: src/app/features/bulk-upload/workplace-references-page/workplace-references-page.component.ts [hunk 1]
```diff
@@ -45,7 +45,11 @@ export class WorkplaceReferencesPageComponent extends BulkUploadReferences {
     }
 
     this.backService.setBackLink({ url: this.exit });
-    this.references = this.activatedRoute.snapshot.data.workplaceReferences;
+    this.references = filter(
+      this.activatedRoute.snapshot.data.workplaceReferences,
+      reference => reference.dataOwner === WorkplaceDataOwner.Parent
+    );
+    console.log(this.references);
```
Comment (omarCreativeDev) [inline src/app/features/bulk-upload/workplace-references-page/workplace-references-page.component.ts]: console log

Comment (tgreyuk) [inline src/app/features/bulk-upload/workplace-references-page/workplace-references-page.component.ts]: console.log

Comment (ChazUK) [reply src/app/features/bulk-upload/workplace-references-page/workplace-references-page.component.ts]: removed

**Comment by ChazUK** [comment]
[FE | Changed information modal is incorrect where BE returns 'not relevant'](https://trello.com/c/L525dgWY/142-fe-changed-information-modal-is-incorrect-where-be-returns-not-relevant)

**Pull request merged**